pennylane/hf/integrals.py

```diff
@@ -308,6 +308,207 @@ def overlap_integral(*args):
     return overlap_integral
 
 
+def _hermite_moment(alpha, beta, t, e, rc):
+    r"""Compute Hermite moment integral recursively.
+
+    The Hermite moment integral in one dimension is defined as
+
+    .. math::
+
+        \int_{-\infty }^{+\infty} x_C^e \Lambda_t dx,
+
+    where :math:`e` is the multipole moment order, :math:`C` is the origin of the Cartesian
```

Comment:
I had used it to define the exponent of the position component but it can be misleading. Mainly inspired by the fact that dipole moment integrals always have e = 1, quadruple integrals might have e = 1 or 2, for example q_xx = \sum_i  Q_i x_i^2.